### PR TITLE
pjsip: add patches for CVE-2022-39269 & CVE-2022-39244

### PR DESCRIPTION
**Pull request by risicle**
###### Description of changes

https://nvd.nist.gov/vuln/detail/CVE-2022-39269
https://nvd.nist.gov/vuln/detail/CVE-2022-39244

Both quite serious.

Also tested with tests from #169498

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by risicle** [comment]
@linsui the addition of these patches breaks jami's custom version of pjsip. The build can be fixed by not including `old.patches` when building the custom version, **but** if we did that we would have to mark jami with `knownVulnerabilities`. It looks like jami upstream don't appear to be on top of this, with their `master` version still shipping 2.12.0, which has additional vulnerabilties.

**Comment by linsui** [comment]
I report it upstream. For now please just mark jami with `knownVulnerabilities`. Thanks!

**Comment by risicle** [comment]
Done. Thanks for the quick response @linsui .

File: pkgs/applications/networking/instant-messengers/jami/default.nix [hunk 1]
```diff
@@ -71,10 +71,14 @@ rec {
         sha256 = "sha256-ENRfQh/HCXqInTV0tu8tGQO7+vTbST6XXpptERXMACE=";
       };
 
-      patches = old.patches ++ (map (x: patch-src + x) (readLinesToList ./config/pjsip_patches));
+      patches = (map (x: patch-src + x) (readLinesToList ./config/pjsip_patches));
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/instant-messengers/jami/default.nix:74]:
```suggestion
      patches = map (x: patch-src + x) (readLinesToList ./config/pjsip_patches);
```

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**